### PR TITLE
Add documentation for `New from clipboard` feature

### PR DESCRIPTION
**Pull request by psobolewskiPhD**
# References and relevant issues
Depends on https://github.com/napari/napari/pull/6532

# Description
This PR adds mentions of the new feature: File menu item `New Image from Clipboard` to the `viewer tutorial` as well as the `getting started` and `quick start` guides.

### CODE REVIEW

File: docs/tutorials/fundamentals/getting_started.md [hunk 1]
```diff
@@ -32,10 +32,8 @@ This command will launch an empty viewer:
 ![image: an empty napari viewer](../assets/tutorials/launch_cli_empty.png)
 
 Once you have the viewer open you can add images through the `File -> Open` dropdown menu
-or by dragging and dropping images directly on the viewer.
-We currently only support files that can be read with [`skimage.io.imread`](https://scikit-image.org/docs/dev/api/skimage.io.html#skimage.io.imread),
-such as `tif`, `png`, and `jpg`.
-We plan on adding support for more exotic file types shortly - see [issue #379](https://github.com/napari/napari/issues/379) for discussion.
+or by dragging and dropping images directly on the viewer. We currently only support files that can be read with [`skimage.io.imread`](https://scikit-image.org/docs/dev/api/skimage.io.html#skimage.io.imread),
+such as `tif`, `png`, and `jpg`.  We plan on adding support for more exotic file types shortly - see [issue #379](https://github.com/napari/napari/issues/379) for discussion. Finally, you can use the `File` menu to make a new `Image` layer from an image in your Clipboard (keybinding {kbd}`Command/Ctrl+N`).
```
Comment (lucyleeow) [inline docs/tutorials/fundamentals/getting_started.md]:
```suggestion
such as `tif`, `png`, and `jpg`.  We plan on adding support for more exotic file types shortly - see [issue #379](https://github.com/napari/napari/issues/379) for discussion. Finally, you can use the `File -> New Image from Clipboard` menu to make a new `Image` layer from an image in your Clipboard (keybinding {kbd}`Command/Ctrl+N`).
```

File: docs/tutorials/fundamentals/quick_start.md [hunk 2]
```diff
@@ -131,6 +131,10 @@ viewer.add_image(data.cell(), name='cell')
 nbscreenshot(viewer, alt_text="image of a single cell opened in napari viewer")
 ```
 
+```{note}
+Once you have a napari viewer open, you can also make a new `Image` layer from an image in your Clipboard using the File menu (keybinding {kbd}`Command/Ctrl+N`).
```
Comment (lucyleeow) [inline docs/tutorials/fundamentals/quick_start.md]:
```suggestion
Once you have a napari viewer open, you can also make a new `Image` layer from an image in your Clipboard using `File -> New Image from Clipboard` menu item (keybinding {kbd}`Command/Ctrl+N`).
```

**Review by lucyleeow — APPROVED** [review]
LGTM!

I don't know why files tab looks like this though:

image

Maybe it's just me?

**Comment by psobolewskiPhD** [comment]
@lucyleeow I have no idea, I see similar on my end. The whole file is weird:
https://github.com/napari/docs/blob/main/docs/tutorials/fundamentals/quick_start.md?plain=1

**Comment by lucyleeow** [comment]
True! I wonder if it has anything to do with the jupytext header (clutching at straws though):

https://github.com/napari/docs/blob/e6d0ad14b7d546ee63c28dde06f2fad54e1be1ab/docs/tutorials/fundamentals/quick_start.md?plain=1#L2-L3

Maybe @melissawm will know why?

**Comment by melissawm** [comment]
> True! I wonder if it has anything to do with the jupytext header (clutching at straws though):
> 
> https://github.com/napari/docs/blob/e6d0ad14b7d546ee63c28dde06f2fad54e1be1ab/docs/tutorials/fundamentals/quick_start.md?plain=1#L2-L3
> 
> Maybe @melissawm will know why?

That may be it. We could probably remove the formats line anyway as we don't need to keep any notebooks synchronized here.

File: docs/tutorials/fundamentals/getting_started.md [hunk 3]
```diff
@@ -34,10 +34,8 @@ This command will launch an empty viewer:
 ![image: an empty napari viewer](../assets/tutorials/launch_cli_empty.png)
 
 Once you have the viewer open you can add images through the `File -> Open` dropdown menu
-or by dragging and dropping images directly on the viewer.
-We currently only support files that can be read with [`skimage.io.imread`](https://scikit-image.org/docs/dev/api/skimage.io.html#skimage.io.imread),
-such as `tif`, `png`, and `jpg`.
-We plan on adding support for more exotic file types shortly - see [issue #379](https://github.com/napari/napari/issues/379) for discussion.
+or by dragging and dropping images directly on the viewer. We currently only support files that can be read with [`skimage.io.imread`](https://scikit-image.org/docs/dev/api/skimage.io.html#skimage.io.imread),
+such as `tif`, `png`, and `jpg`.  We plan on adding support for more exotic file types shortly - see [issue #379](https://github.com/napari/napari/issues/379) for discussion. Finally, you can use the `File -> New Image from Clipboard` menu item to make a new `Image` layer from an image (or URL to an image) in your Clipboard (keybinding {kbd}`Command/Ctrl+N`).
```
Comment (melonora) [inline docs/tutorials/fundamentals/getting_started.md]: Should we also make mention here of `File -> Open with plugin`?

Comment (psobolewskiPhD) [reply docs/tutorials/fundamentals/getting_started.md]: I think that's not related to the feature being documented here?
And frankly I don't know what that menu item does because normal File > Open File uses plugins to open...
Frankly, I think we need an `opening files` guide/tutorial whatever that documents what files can be opened by napari and that other files can be opened by plugins.

Comment (lucyleeow) [reply docs/tutorials/fundamentals/getting_started.md]: >  And frankly I don't know what that menu item does because normal File > Open File uses plugins to open...

I think that difference is explained here: https://github.com/napari/napari/pull/4347
tl;dr - open tries to open with the preferred plugin from settings and open with plugin forces popping up of dialogue to choose plugin to open with. Practically, they both call `_open_files_dialog`, one just gives `choose_plugin=True` arg.

Comment (psobolewskiPhD) [reply docs/tutorials/fundamentals/getting_started.md]: Definitely think `File > Open with plugin` should get proper documentation. I added that to the existing issue:
https://github.com/napari/docs/issues/44
I think it's outside the scope of this PR which documents a specific feature recently added.

In general the opening/saving options of vanilla napari and key plugins needs dedicated docs.

Comment (GenevieveBuckley) [inline docs/tutorials/fundamentals/getting_started.md]:
```suggestion
such as `tif`, `png`, and `jpg`.  We plan on adding support for more exotic file types shortly - see [issue #379](https://github.com/napari/napari/issues/379) for discussion. Finally, you can use the `File -> New Image from Clipboard` menu item to make a new `Image` layer from an image (or URL to an image) copied to your Clipboard (keybinding {kbd}`Command/Ctrl+N`).
```

File: docs/tutorials/fundamentals/quick_start.md [hunk 4]
```diff
@@ -131,6 +131,10 @@ viewer.add_image(data.cell(), name='cell')
 nbscreenshot(viewer, alt_text="image of a single cell opened in napari viewer")
 ```
 
+```{note}
+Once you have a napari viewer open, you can also make a new `Image` layer from an image (or URL to an image) in your Clipboard using `File -> New Image from Clipboard` menu item (keybinding {kbd}`Command/Ctrl+N`).
```
Comment (GenevieveBuckley) [inline docs/tutorials/fundamentals/quick_start.md]:
```suggestion
Once you have a napari viewer open, you can also make a new `Image` layer from an image (or URL to an image) copied to your Clipboard using `File -> New Image from Clipboard` menu item (keybinding {kbd}`Command/Ctrl+N`).
```

File: docs/tutorials/fundamentals/viewer.md [hunk 5]
```diff
@@ -108,7 +108,7 @@ We'll go through each of these in the next sections.
 
 The main menu consists of the **File**, **View**, **Window**, **Plugins**, and **Help** options.
 
-* **File** has the options to open files, folders, and samples, save layers and screenshots,copy screenshots to clipboard and, in the Windows version, preferences.
+* **File** has the options to open files, folders, and samples, save layers and screenshots, copy screenshots to clipboard and, in the Windows version, preferences. Additionally, you can make a new `Image` layer from an image (or URL to an image) in your Clipboard (keybinding {kbd}`Command/Ctrl+N`).
```
Comment (GenevieveBuckley) [inline docs/tutorials/fundamentals/viewer.md]:
```suggestion
* **File** has the options to open files, folders, and samples, save layers and screenshots, copy screenshots to clipboard and, in the Windows version, preferences. Additionally, you can make a new `Image` layer from an image (or URL to an image) copied to your Clipboard (keybinding {kbd}`Command/Ctrl+N`).
```

**Review by GenevieveBuckley — APPROVED** [review]
This is great, my only suggestion is to change the phrasing from "an image in your Clipboard" to "an image copied to your Clipboard".

I think it's useful to have the word "copy" in the sentence since we have a lot of non-native English speakers who use napari. I think it makes it a bit easier for people to remember that "Clipboard" is related to copy-paste actions if we provide the extra context clue.

**Comment by psobolewskiPhD** [comment]
@GenevieveBuckley Excellent point, thanks for that. I've committed the suggestions!
❤️
